baybe/searchspace/core.py

```diff
@@ -71,10 +71,16 @@ class SearchSpace(SerialMixin):
     def __str__(self) -> str:
         start_bold = "\033[1m"
         end_bold = "\033[0m"
-        searchspace_str = f"""{start_bold}Search Space{end_bold}
-        \n{start_bold}Search Space Type: {end_bold}{self.type.name}
-        \n{self.discrete}
-        \n{self.continuous}"""
+        head_str = f"""{start_bold}Search Space{end_bold}
+        \n{start_bold}Search Space Type: {end_bold}{self.type.name}"""
+
+        # Check the sub space size to avoid adding unwanted break lines
+        # if the sub space is empty
+        discrete_str = f"\n\n{self.discrete}" if len(str(self.discrete)) > 0 else ""
```

Comment:
Shouldn't you be able to use `self.discrete.is_empty` (or however that is called) here?

Reply:
Indeed it is. I fixed it, thank you Alex!